src/common/utils.ts

```diff
@@ -4,3 +4,9 @@ import lclStorage from 'store/storages/localStorage';
 export const store = engine.createStore([lclStorage]);
 
 export const dedupe = (array: string[]) => [...new Set(array)];
+
+export const truncateMiddle = (input: string, offset: number) => {
+  const start = input.substr(0, offset);
+  const end = input.substr(input.length - offset, input.length);
+  return `${start}…${end}`;
```

Comment:
Should we use `&hellip;`?

Reply:
Not to the best of my knowledge, what's wrong with using the unicode character directly?

Reply:
I think if we use the unicode, it won't break the line if there is a situation where it would break: eg

```
asdas..
.asda
```

Reply:
I had a quick google but can't find anywhere suggesting you shouldn't use unicode characters in JSX. I thought the days of `&amp;` and `&hellip;` were past us.